src/actions/view.jsx

```diff
@@ -43,6 +46,30 @@ export function printModeChanged() {
   }
 }
 
+/**
+ * Notifies the application that the print mode has changed
+ *
+ * @returns {string} a packaged payload to be used by Redux reducers
+ */
+export function printModeOn() {
+  return {
+    type: PRINT_MODE_ON,
+    requery: REQUERY_NEVER
+  }
+}
+
+/**
+ * Notifies the application that the print mode has changed
+ *
+ * @returns {string} a packaged payload to be used by Redux reducers
+ */
+export function printModeOff() {
+  return {
+    type: PRINT_MODE_OFF,
+    requery: REQUERY_NEVER
+  }
+}
+
```

Comment:
I prefer the separate methods rather than `toggle` because of [idempotency](https://en.wikipedia.org/wiki/Idempotence#Computer_science_meaning)

printModeOff x N = printMode: Off

printModeToggle x N = ????

Reply:
definitely easier to test and we know what to expect. 👍